docs/architecture/adr-070-unordered-account.md

```diff
@@ -86,32 +86,20 @@ var (
 	unorderedSequencePrefix = collections.NewPrefix(90)
 )
 
-type UnorderedTxManager struct {
+type AccountKeeper struct {
+	// ...
 	unorderedSequences collections.KeySet[collections.Pair[uint64, []byte]]
 }
 
-func NewUnorderedTxManager(kvStore store.KVStoreService) *UnorderedTxManager {
-	sb := collections.NewSchemaBuilder(kvStore)
-	m := &UnorderedTxManager{
-		unorderedSequences: collections.NewKeySet(
-			sb,
-			unorderedSequencePrefix,
-			"unordered_sequences",
-			collections.PairKeyCodec(collections.Uint64Key, collections.BytesKey),
-		),
-	}
-	return m
-}
-
-func (m *UnorderedTxManager) Contains(ctx sdk.Context, sender []byte, timestamp uint64) (bool, error) {
+func (m *AccountKeeper) Contains(ctx sdk.Context, sender []byte, timestamp uint64) (bool, error) {
 	return m.unorderedSequences.Has(ctx, collections.Join(timestamp, sender))
 }
 
-func (m *UnorderedTxManager) Add(ctx sdk.Context, sender []byte, timestamp uint64) error {
+func (m *AccountKeeper) Add(ctx sdk.Context, sender []byte, timestamp uint64) error {
 	return m.unorderedSequences.Set(ctx, collections.Join(timestamp, sender))
 }
 
-func (m *UnorderedTxManager) RemoveExpired(ctx sdk.Context) error {
+func (m *AccountKeeper) RemoveExpired(ctx sdk.Context) error {
```

Comment:
I would just suggest something a little more detailed than simply `Contains`, `Add`, etc. that mentions unordered